sky/resources.py

```diff
@@ -1293,6 +1293,10 @@ def get_reservations_available_resources(self) -> Dict[str, int]:
             skypilot_config.get_nested(
                 (str(self.cloud).lower(), 'specific_reservations'), set()))
 
+        if isinstance(self.cloud, clouds.DummyCloud):
+            return self.cloud.get_reservations_available_resources(
+                '', '', None, specific_reservations)
+
         assert (self.cloud is not None and self.instance_type is not None and
                 self.region
                 is not None), ('Cloud, instance type, region must be specified')
```

Comment:
let's include the `self` and the variables in the assertion output as well to make the future debugging easier.